<launch>

	<node pkg="course_agv_slam_task" type="particle_filter" name="particle_filter" output="screen">
		
		<!--from a rosbag-->
        <param name="init_x" value="0"/>
        <param name="init_y" value="0"/>
        <param name="init_theta" value="0"/>

        <param name="init_rand_xy" value="0.5"/>
        <param name="init_rand_theta" value="0.3"/>

        <param name="N_th" value="8000"/>

        <param name="z_hit" value="0.75"/>
        <param name="std_dev" value="0.25"/>

        <param name="ratio" value="0.9"/>
	</node>

    <include file="$(find course_agv_slam_task)/launch/extraction.launch"></include>
    <include file="$(find course_agv_slam_task)/launch/icp_lm.launch"></include>
</launch>
